### DISCUSSION TITLE
PayPal - Add settings so user can disable fundings

### DISCUSSION THREAD

**Discussion by philipboomy** [craftcms/commerce#2580] [Ideas] [answered]
Add settings to PayPal plugin so user can disable different payment funding. See here https://developer.paypal.com/docs/checkout/reference/customize-sdk/#disable-funding

**Comment by nfourtythree** [comment, marked as answer]
This feature is available in the [Commerece PayPal Checkout](https://github.com/craftcms/commerce-paypal-checkout) plugin.

You can see the feature request and resolution here: https://github.com/craftcms/commerce-paypal-checkout/issues/14

Thanks!
